### ISSUE TITLE
Try the new user flow, starting from scratch

### ISSUE THREAD

**Issue by cinnamon-bun** [fraction/oasis#274]
**What's the problem you want solved?**
Can a new person join SSB using only Oasis?

I've never tried this so I'm not actually sure if it's possible.

**Is there a solution you'd like to recommend?**
Try it

Look for rough edges.  Do we need to show instructions when starting up with a fresh and empty account?

**Comment by christianbundy** [comment]
I've done this on Android when experimenting with Termux, and the problems I saw were:

- Need #208 
- #237 is confusing
- I think SSB-CONN avoids connecting to peers unless you give it explicit permission, which means that even if you know about peers I think you only peer with people you follow directly.

**Comment by christianbundy** [comment]
I should mention that the above attempt was when I was on the LAN with my main feed. I've never tried the pub-only setup but I imagine it's similar.

**Comment by georgeowell** [comment]
Going to test this again on desktop.

**Comment by murilopolese** [comment]
Should I try with a specific version of Oasis or any branch in specific?

**Comment by zelfos** [comment]
I've created a new identity from desktop using the latest stable version (2.16.0) with the help of a pub invitation. Everything is looking fine, only a little slow since I'm only following the pub and my main account.

**Comment by christianbundy** [comment]
@murilopolese Whichever you like! There should be instructions in the readme for the stable version (up to ~28 days old) or the unstable version (more recent but more turbulent).

**Comment by murilopolese** [comment]
I'm a very clumsy person with files so I'm not going to play with my current ssb setup. I did install it on my host computer and it works really fine so far together with Patchwork. For the test I thought it would be straight forward to setup with Docker but I'm having a couple of issues to even run the app. Should I open an issue for that perhaps? I believe it's going to be much easier to test this workflow having an isolated container :s

**Comment by christianbundy** [comment]
Yeah, please open an issue if you have snags with the Docker setup!

**Comment by murilopolese** [comment]
I have tried for a week to start a new identity from scratch on Oasis and I'm not sure whether the problem is with Oasis, with my setup or with the solutions I picked.

In general I think my main struggle is when I click on "connect and sync". The request takes too long and the browser resets the connection. I tried to create both a pub and a room with `ssb-room` so I could have a easier start and in fact I can connect to the room with other clients (Patchwork and Manyverse) but not with oasis.

Also, it took me a little while to figure out what the peer connection buttons actually do, I think I understood but still mysterious.

My next attempt is to drop a prepopulated `conn.json` on the `.ssb` folder inside the container and hope for no explosions.

**Comment by christianbundy** [comment]
> In general I think my main struggle is when I click on "connect and sync".

Oh, I could see why that would look appealing when you're syncing, but would actually never complete if you're not on the network (cc: @nickwynja). I don't think Oasis supports SSB-Room invites, just pub invites right now. Were you able to get up and running with a pub invite? The flow should be:

- Start Oasis
- Select 'Settings'
- Paste your pub invite code
- Select 'Accept invite' or whatever the button says
- Select 'Latest'
- Watch the messages pour in

**Comment by murilopolese** [comment]
Right! This explains it. I'll try to start the pub again but I did manage
to connect to a pub before so it shouldn't be a big problem.

**Comment by aadilayub** [comment]
I tried setting up a new work profile on my laptop today. Is there any way to connect to the network without using a pub? I tried using a peer invite from my manyverse account, but it seems that isn't supported.

in short: is it possible to directly follow my other accounts without connecting to a pub first?

**Comment by christianbundy** [comment]
@aadilayub You can type their ID into the search, which should take you to their profile and let you follow them, but unless you've copied a `conn.json` you won't know of any peers that you can connect to so that you can replicate messages.

**Comment by cbleslie** [comment]
@christianbundy, Nicety. I think. This might be a distraction, if the goal is to migrate users from patchwork. This would be perfect for a milestone that is focused on polishing user experience and documentation.
